cla_frontend/settings/docker.py

```diff
@@ -8,7 +8,7 @@
     SECRET_KEY = "CHANGE_ME"
     pass
 
-DEBUG = True if os.environ.get('SET_DEBUG') == 'True' else False
+DEBUG = True if os.environ.get('DEBUG') == 'True' else False
```

Comment:
This should probably be
```
DEBUG = os.environ.get('DEBUG', 'False') == 'True'
````
to make sure it doesn't error if the env variable is not set.

Reply:
👍  Good point